packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -304,26 +307,18 @@ export default class TradeStore extends BaseStore {
         if (error) {
             showError({ message: localize('Trading is unavailable at this time.') });
             return;
-        } else if (!active_symbols || !active_symbols.length) {
+        }
+
+        if (!active_symbols || !active_symbols.length) {
             await WS.wait('get_settings');
-            if (
-                can_have_mlt_or_mx_account &&
-                is_logged_in &&
-                !localStorage.getItem('hide_close_mx_mlt_account_notification')
-            ) {
+            if (can_have_mlt_or_mx_account && is_logged_in && !hide_close_mx_mlt_storage_flag) {
                 setError(true, {
                     type: 'mx_mlt_removal',
                 });
-            } else if (is_logged_in && localStorage.getItem('hide_close_mx_mlt_account_notification')) {
-                if (can_have_mlt_account) {
-                    showMxMltUnavailableError(showError, can_have_mlt_account);
-                    return;
-                } else if (can_have_mx_account) {
-                    showMxMltUnavailableError(showError, false, can_have_mx_account);
-                    return;
-                }
```

Comment:
This was the problematic part. the flow would get into `else` which was not there. I've merged both conditions into 1.